worlds/animal_well/options.py

```diff
@@ -132,14 +132,49 @@ class WheelTricks(Choice):
     default = 0
 
 
-class WeirdTricks(Toggle):
+class BallThrowing(Choice):
     """
-    Include performing "weird" tricks in the logic.
-    Some of these tricks are difficult, tedious, or inconsistent.
-    Use at your own risk.
+    Include in logic using the ball to hit switches or buttons not ""designed"" for it.
+    Off means the ball will rarely be used for anything other than breaking blocks, spikes or guard shields.
+    Simple means the ball can be used to hit easy targets without any real rebound, setup or moving. Most ""yoyo chute"" buttons are included here.
+    Advanced means hitting your target may require bouncing off a wall, or moving while throwing to adjust momentum or get a good angle.
+    Expert includes any more complicated tricks, including those that require specific setups or getting lucky.
```

Comment:
```suggestion
    Include using the ball to hit switches or buttons not "designed" for it in logic.
    Off means the ball will rarely be used for anything other than breaking blocks, spikes, guard shields.
    Simple means the ball can be used to hit easy targets without any real rebound, setup, or movement. Most "yoyo chute" buttons are included here.
    Advanced means hitting your target may require bouncing off a wall or moving while throwing to adjust momentum/angle.
    Expert includes more complicated tricks including those that require specific setups or getting lucky.
```

Reply:
Where did the "or" go on line 138?

Reply:
it got stolen, rip
(you should add it back)